exercises/leibniz-formula/description.md

```diff
@@ -0,0 +1,29 @@
+# Description
+
+Calculate π using [Leibniz formula](https://en.wikipedia.org/wiki/Leibniz_formula_for_%CF%80).
+
+```
+1 - 1/3 + 1/5 - 1/7 + 1/9 - ... = π/4
+```
+
+According to [Gottfried Leibniz](https://en.wikipedia.org/wiki/Gottfried_Leibniz), you can calculate π using the formula
+above.
+The more fractions you use, the closer your calculation will come to π.
+
+Here are some examples:
+
+| number of fractions | calculated π          |
+|---------------------|-----------------------|
+| 0                   | 4                     |
+| 1                   | 2.666666666666666...  |
+| 10                  | 3.232315809405594...  |
+| 100                 | 3.1514934010709914... |
+| 1 000               | 3.1425916543395442... |
```

Comment:
Some languages have adequate support for fractions. I thought it might be nice to provide (some) fractions in this table, but they quickly become unwieldy:
terms | faction
--- | ---
0 | 0
1 | 4 / 1
2 | 8 / 3
3 | 52 / 15
5 | 1052 / 315
10 | 44257352 / 14549535
50 | 3400605476464206445954873476681150352328 / 1089380862964257455695840764614254743075